components/switch/package.json

```diff
@@ -16,11 +16,11 @@
     "build": "gulp"
   },
   "peerDependencies": {
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/tokens": "^1.0.3"
   },
   "devDependencies": {
-    "@spectrum-css/component-builder": "^3.2.0",
-    "@spectrum-css/vars": "^8.0.0",
+    "@spectrum-css/component-builder-simple": "^1.0.0-beta.0",
+    "@spectrum-css/tokens": "^1.0.3",
```

Comment:
Same as above: `1.0.7`

Reply:
I don't think it's released yet @pfulton . I am getting on a `yarn install`
```
error Couldn't find package "@spectrum-css/tokens@^1.0.7" required by "@spectrum-css/switch@1.0.23" on the "npm" registry.